samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -16,13 +16,16 @@ The sample supports the following development kits:
 
 .. table-from-rows:: /includes/sample_board_rows.txt
    :header: heading
-   :rows: nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf21540dk_nrf52840
+   :rows: nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns, nrf52840dk_nrf52840, nrf52dk_nrf52832, nrf52833dk_nrf52833, nrf52833dk_nrf52820, thingy53_nrf5340_cpuapp_and_cpuapp_ns, nrf21540dk_nrf52840
```

Comment:
Did you check `nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns`?

Reply:
Yes, as far as I can see it works fine.

Reply:
I will re-check it. Im a little uncertain if I checked it. I checked both for thingy at least.

Reply:
It is possible to use the nrf5340dk_nrf5340_cpuapp_and_cpuapp_ns in the README file.

Reply:
Yep, the initial task was about Thingy. I was a bit surprised you added both :)

Reply:
You will get both in the same row instead of having each of them in separate rows.